birdhouse/default.env

```diff
@@ -10,6 +10,9 @@ export BASH_IMAGE="bash:5.1.4"
 
 # Root directory under which all data persistence should be nested under
 export DATA_PERSIST_ROOT="/data"
+export STAC_DATA_LOCATION_FOR_THREDDS_ON_HOST=${DATA_PERSIST_ROOT}/ncml  # this default is for backward compatibility 
```

Comment:
Also they should be added to the `DELAYED_EVAL` variable for the same reason (if `DATA_PERSIST_ROOT` is updated)